spark-connector/README.md

```diff
@@ -22,8 +22,10 @@ $SPARK_HOME/bin/spark-submit ./target/kdp-spark-example.jar --master local[4]
 
 ### KDP Count
 
-This example, in Java, covers the main functionality of the KDP Spark Connector. It includes read, write, and simple ABAC. You will need Spark 3, Maven 3, and Java 8 or Java 11 to run this example.
+This example, in Java, covers the main functionality of the KDP Spark Connector. It includes read, write, and simple ABAC. You will need Spark 3, Maven 3, and Java 8 11 to build and run this example.
```

Comment:
Is this indicating Java 8.11 or Java 8 and/or 11?

Reply:
It should be Java 8 or 11. I've fixed it - thanks for catching that.